### DISCUSSION TITLE
Parameters inside the thalamic model

### DISCUSSION THREAD

**Discussion by HenryMarsden** [neurolib-dev/neurolib#207] [Q&A] [answered]
Hi,

I was trying to understand the equations for calculating change in membrane potential between runs and was looking for some help in understanding some of the parameters.
![image](https://user-images.githubusercontent.com/59068823/168672666-4a441634-b5a9-4bfa-887a-a79bb71c8829.png)
The equations above are the ones I was wanting some help understanding. I understand the numerator of the equation pretty well I think, however what does the denominator represent. By my understanding it should be either conductance or capacitance to result in potential. however I am not sure what tau represents. Also I assume C_m refers to membrane capacitance however why is 1 being divided by it before subtracting the result from tau? And as a final question, why is the result this being multiplied by these specific currents?

I'm sorry if these question are basic but I couldn't find any information in the docs.

Thanks,
Henry Marsden

**Comment by jajcayn** [comment, marked as answer]
Hey @HenryMarsden. The thalamic model is featured in my latest paper here> https://www.frontiersin.org/articles/10.3389/fncom.2022.769860/full. There you can find a whole section dealing with thalamic mass model and detailed explanation what do the equations mean. The core of the model is taken from https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1005022

**Reply by HenryMarsden** [reply]
Thank you very much!
